### PR TITLE
Add labels for wg-batch

### PR DESCRIPTION
**Pull request by mrbobbytables**
Follow up item for the creation of the wg.

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -2049,3 +2055,4 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+
```
Comment (endocrimes) [inline label_sync/labels.yaml:2058]: nit: extra line, shouldn't break anything though.

**Review by endocrimes — APPROVED** [review]
/lgtm

**Review by cblecker — APPROVED** [review]

**Pull request merged**